src/index.js

```diff
@@ -31,7 +31,7 @@ const Root = ({store}) => {
 					<Route path = '/' component={App}>
 						<IndexRoute component={Home} />
 						<Route path = '/newUser' component={NewUser} />
```

Comment:
Perhaps you should change the newUser component also. 
We should make a ticket to change the routes to match components.